The CI issue is odd, but might be related:
```
2023-09-12T15:45:16.6001505Z [INFO] --- exec-maven-plugin:3.0.0:exec (assemble-downstream-doc) @ quarkus-documentation ---
2023-09-12T15:45:17.9662991Z Sep 12, 2023 3:45:17 PM io.quarkus.docs.generation.AssembleDownstreamDocumentation main
2023-09-12T15:45:17.9664895Z ERROR: Unable to read generated file ../target/asciidoc/generated/config/quarkus-log-logging-log-config.adoc
2023-09-12T15:45:18.0107264Z ##[error]Exception in thread "main" java.io.UncheckedIOException: An error occurred while generating the downstream tree
2023-09-12T15:45:18.0278490Z 	at io.quarkus.docs.generation.AssembleDownstreamDocumentation.main(AssembleDownstreamDocumentation.java:189)
2023-09-12T15:45:18.0281913Z Caused by: java.nio.file.NoSuchFileException: ../target/asciidoc/generated/config/quarkus-log-logging-log-config.adoc
2023-09-12T15:45:18.0282617Z 	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
2023-09-12T15:45:18.0283531Z 	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
2023-09-12T15:45:18.0304886Z 	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
2023-09-12T15:45:18.0305599Z 	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
2023-09-12T15:45:18.0306161Z 	at java.base/java.nio.file.Files.newByteChannel(Files.java:371)
2023-09-12T15:45:18.0306619Z 	at java.base/java.nio.file.Files.newByteChannel(Files.java:422)
2023-09-12T15:45:18.0307179Z 	at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:420)
2023-09-12T15:45:18.0307726Z 	at java.base/java.nio.file.Files.newInputStream(Files.java:156)
2023-09-12T15:45:18.0308206Z 	at java.base/java.nio.file.Files.newBufferedReader(Files.java:2839)
2023-09-12T15:45:18.0326020Z 	at java.base/java.nio.file.Files.readAllLines(Files.java:3330)
2023-09-12T15:45:18.0326498Z 	at java.base/java.nio.file.Files.readAllLines(Files.java:3370)
2023-09-12T15:45:18.0327174Z 	at io.quarkus.docs.generation.AssembleDownstreamDocumentation.copyAsciidoc(AssembleDownstreamDocumentation.java:266)
2023-09-12T15:45:18.0328021Z 	at io.quarkus.docs.generation.AssembleDownstreamDocumentation.main(AssembleDownstreamDocumentation.java:167)
2023-09-12T15:45:18.0363788Z [ERROR] Command execution failed.
2023-09-12T15:45:18.0366136Z org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
2023-09-12T15:45:18.0366812Z     at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
2023-09-12T15:45:18.0368036Z     at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
2023-09-12T15:45:18.0369063Z     at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:982)
2023-09-12T15:45:18.0369657Z     at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:929)
2023-09-12T15:45:18.0372024Z     at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:457)
2023-09-12T15:45:18.0378164Z     at com.gradle.maven.cache.extension.d.c.a (SourceFile:27)
2023-09-12T15:45:18.0381606Z     at com.gradle.maven.cache.extension.d.n.a (SourceFile:23)
2023-09-12T15:45:18.0382915Z     at com.gradle.maven.cache.extension.d.h.a (SourceFile:28)
2023-09-12T15:45:18.0419806Z     at com.gradle.maven.cache.extension.d.m.a (SourceFile:27)
2023-09-12T15:45:18.0420407Z     at com.gradle.maven.cache.extension.d.a.c (SourceFile:115)
2023-09-12T15:45:18.0420874Z     at com.gradle.maven.cache.extension.d.a.a (SourceFile:61)
2023-09-12T15:45:18.0421329Z     at com.gradle.maven.cache.extension.d.e.a (SourceFile:27)
2023-09-12T15:45:18.0421782Z     at com.gradle.maven.cache.extension.d.l.a (SourceFile:17)
2023-09-12T15:45:18.0424024Z     at com.gradle.maven.cache.extension.d.d.a (SourceFile:42)
2023-09-12T15:45:18.0424618Z     at com.gradle.maven.cache.extension.d.b.a (SourceFile:26)
2023-09-12T15:45:18.0425074Z     at com.gradle.maven.cache.extension.d.f$1.run (SourceFile:35)
2023-09-12T15:45:18.0425960Z     at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
2023-09-12T15:45:18.0427224Z     at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
2023-09-12T15:45:18.0428286Z     at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
2023-09-12T15:45:18.0428919Z     at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
2023-09-12T15:45:18.0429577Z     at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:157)
2023-09-12T15:45:18.0430429Z     at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
2023-09-12T15:45:18.0431080Z     at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
2023-09-12T15:45:18.0432350Z     at com.gradle.maven.cache.extension.d.f.a (SourceFile:31)
2023-09-12T15:45:18.0432810Z     at com.gradle.maven.cache.extension.d.k.a (SourceFile:67)
2023-09-12T15:45:18.0433295Z     at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:79)
2023-09-12T15:45:18.0433765Z     at com.sun.proxy.$Proxy140.execute (Unknown Source)
2023-09-12T15:45:18.0434268Z     at com.gradle.maven.scan.extension.internal.d.a.executeMojo (SourceFile:114)
2023-09-12T15:45:18.0435375Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:370)
2023-09-12T15:45:18.0436148Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
2023-09-12T15:45:18.0436735Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
2023-09-12T15:45:18.0437742Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
2023-09-12T15:45:18.0439096Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
2023-09-12T15:45:18.0439885Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2023-09-12T15:45:18.0440705Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2023-09-12T15:45:18.0519706Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2023-09-12T15:45:18.0520642Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
2023-09-12T15:45:18.0521252Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:299)
2023-09-12T15:45:18.0522295Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:193)
2023-09-12T15:45:18.0522781Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:106)
2023-09-12T15:45:18.0523263Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:963)
2023-09-12T15:45:18.0523736Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:296)
2023-09-12T15:45:18.0524396Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:199)
2023-09-12T15:45:18.0525080Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2023-09-12T15:45:18.0526200Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2023-09-12T15:45:18.0527006Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2023-09-12T15:45:18.0527914Z     at java.lang.reflect.Method.invoke (Method.java:566)
2023-09-12T15:45:18.0528470Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2023-09-12T15:45:18.0529121Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2023-09-12T15:45:18.0529779Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2023-09-12T15:45:18.0530823Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2023-09-12T15:45:18.0531958Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2023-09-12T15:45:18.0532749Z     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2023-09-12T15:45:18.0534101Z     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2023-09-12T15:45:18.0535137Z     at java.lang.reflect.Method.invoke (Method.java:566)
2023-09-12T15:45:18.0535664Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
2023-09-12T15:45:18.0536446Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
2023-09-12T15:45:18.0537035Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
```

--- a/docs/src/main/asciidoc/logging.adoc
+++ b/docs/src/main/asciidoc/logging.adoc
@@ -247,7 +247,8 @@ A category configuration recursively applies to all subcategories of that catego
 The parent of all logging categories is called the "root category".
 This category, being the ultimate parent, may contain configuration which applies globally to all other categories. This includes the globally configured handlers and formatters.
 
-Thus, configurations made under `quarkus.log.console.*`, `quarkus.log.file.*`, and `quarkus.log.syslog.*`, are global and apply for all categories. For more information, see <<loggingConfigurationReference>>.
+Thus, configurations made under `quarkus.log.console.+*+`, `quarkus.log.file.+*+`, and `quarkus.log.syslog.+*+` are global and apply for all categories.
+For more information, see <<loggingConfigurationReference>>.
 
 If you want to configure something extra for a specific category, create a named handler like `quarkus.log.handler.[console|file|syslog].<your-handler-name>.*` and set it up for that category by using `quarkus.log.category.<my-category>.handlers`.
 

